pkg/gds/gds.go

```diff
@@ -322,6 +336,47 @@ func (s *GDS) Register(ctx context.Context, in *api.RegisterRequest) (out *api.R
 	return out, nil
 }
 
+func (s *GDS) beginRegistration(ctx context.Context, vasp *pb.VASP, contactEmail string) (err error) {
```

Comment:
Can we add a docstring here which summarizes what this method does? Specifically as a maintainer of GDS I would want to know (without having to look at the method):

1. Does this method update the VASP in the database or require the caller to do it?
2. Why is contactEmail required to be passed in and what email does it refer to?
3. Does this method return a normal error or a gRPC error? (e.g. does the caller have to create a gRPC status error or just return the error?)